.github/workflows/snap.yaml

```diff
@@ -0,0 +1,32 @@
+name: snap
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
This means that this workflow is going to be triggered by any tag on any branch. I think we only want the confined one here.